main/main.go

```diff
@@ -175,8 +176,8 @@ func getK8sAndIstioCVEs(feedReader map[string]string) ([]utils.NvdCVEEntry, []ut
 	return allK8sCVEs, allIstioCVEs, nil
 }
 
-func filterCVEs(cves *utils.NvdCVEs, project string) []utils.NvdCVEEntry {
-	var cveEntries []utils.NvdCVEEntry
+func filterCVEs(cves *nvd.CVEs, project string) []nvd.CVEEntry {
```

Comment:
If you're moving it out into a separate package, then it feels like this function (and the only below) should be pushed into that package too (maybe as a member function of the `nvd.CVEs` struct)